plugins/com.google.cloud.tools.eclipse.ui.status/OSGI-INF/polling.xml

```diff
@@ -3,6 +3,6 @@
    <service>
       <provide interface="com.google.cloud.tools.eclipse.ui.status.GcpStatusMonitoringService"/>
    </service>
-   <reference bind="setProxyService" cardinality="0..1" interface="org.eclipse.core.net.proxy.IProxyService" name="ProxyService" policy="dynamic" unbind="unsetProxyService"/>
+   <reference bind="setProxyService" cardinality="0..1" interface="IProxyService" name="ProxyService" policy="dynamic" unbind="unsetProxyService"/>
```

Comment:
Should we revert this? My Eclipse restore this to `org.eclipse.core.net.proxy.IProxyService` automatically.

Reply:
And my Eclipse moved it. I don't know why, either way.

Reply:
Should be reverted: the interface is a fully-qualified type name.